Trepang2/Trepang2.asl

```diff
@@ -266,6 +266,13 @@ init
             var ABaseGameMode_C_CurrentCutscene = getProperty(ABaseGameMode_C, "CurrentCutscene");
             var ABaseGameMode_C_CurrentCutscene_Offset = getPropertyOffset(ABaseGameMode_C_CurrentCutscene);
             vars.Log("CurrentCutscene Offset: " + ABaseGameMode_C_CurrentCutscene_Offset.ToString("X"));
+
+            var ABaseGameMode_C_AllLiveBaseChars = getProperty(ABaseGameMode_C, "AllLiveBaseChars");
+            var ABaseGameMode_C_AllLiveBaseCharsArrayPtr_Offset = getPropertyOffset(ABaseGameMode_C_AllLiveBaseChars);
+            vars.Log("AllLiveBaseCharsArrayPtr Offset: " + ABaseGameMode_C_AllLiveBaseCharsArrayPtr_Offset.ToString("X"));
+            var ABaseGameMode_C_AllLiveBaseCharsArrayCount_Offset = getPropertyOffset(ABaseGameMode_C_AllLiveBaseChars) + 8;
+            vars.Log("AllLiveBaseCharsArrayCount Offset: " + ABaseGameMode_C_AllLiveBaseCharsArrayCount_Offset.ToString("X"));
```

Comment:
I'm quite excited someone else was able to use this stuff without my help :D nice stuff